index.rst

```diff
@@ -109,66 +137,54 @@ syntactically equivalent to a semantic version
 appended.
 
 These are **not** actually semantic versions, in that the compatibility
-guarantees of true semantic versions are not present.  The version
-numbers are constructed purely to aid sortability and version choosing,
-because there are excellent extant tools to manipulate semantic
-versions.
-
-*Within* a given release type for a given image,
-semantic versions (if supported for that type) can be used to compare
-two versions of the Lab container: a higher number is more recent.
-Images are
-comparable *only within a tag type*.  That is, the semantic versions
-from (e.g.) a weekly and a release image are not comparable.
+guarantees of true semantic versions are not present.
+The version numbers are constructed purely to aid sortability and version choosing, because there are excellent extant tools to manipulate semantic versions.
+
+*Within* a given release type for a given image, semantic versions (if supported for that type) can be used to compare two versions of the Lab container: a higher number is more recent.
+Images are comparable *only within a tag type*.
+That is, the semantic versions from (e.g.) a weekly and a release image are not comparable.
 
 The ``cycle`` and ``rest`` fields end up within the build field of a
-semantic version.  The information in ``cycle`` is already
-semver-compatible, and ``rest`` is transformed as follows: underscores
-become dots, and any nonalphanumeric characters remaining are simply
-dropped.
+semantic version.
+The information in ``cycle`` is already semver-compatible, and ``rest`` is transformed as follows: underscores become dots, and any nonalphanumeric characters remaining are simply dropped.
```

Comment:
I think it's a bit deceptive to call this "semver-compatible" since I don't think we treat the cycle as a semver at all.

Reply:
What it means is that the cycle format is acceptable to semver as part of the build tag (unlike "rest", we never have to transform it to semver-legal characters)